src/gulpfile.babel.js

```diff
@@ -99,22 +102,22 @@ const reload = done => {
 };
 
 /**
- * Task: `styles`.
- *
- * Compiles Sass, Autoprefixes it and Minifies CSS.
- *
- * This task does the following:
- *    1. Gets the source scss file
- *    2. Compiles Sass to CSS
- *    3. Writes Sourcemaps for it
- *    4. Autoprefixes it and generates style.css
+ * This function does the following:
+ *    1. Compiles Sass to CSS
+ *    2. Writes Sourcemaps for it
+ *    3. Autoprefixes it and generates .css
+ *    4. Renames the CSS file with suffix -rtl and generates -rtl.css
  *    5. Renames the CSS file with suffix .min.css
- *    6. Minifies the CSS file and generates style.min.css
+ *    6. Minifies the CSS file and generates .min.css
  *    7. Injects CSS or reloads the browser via browserSync
  */
-gulp.task( 'styles', () => {
-	return gulp
-		.src( config.styleSRC, { allowEmpty: true })
+function processStyle( gulpStream, processOptions = {} ) {
+	processOptions = defaults( processOptions, {
+		isRTL: false,
+		styleDestination: './',
```

Comment:
Even with simple styles the `styleDestination` should always be provided for folks who change the location of it for some reason.